gpii/node_modules/settingsHandlers/test/WebSocketsSettingsHandlerTests.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * WebSockets Settings Handler Tests
+ *
+ * Copyright 2013, 2014 Emergya
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013)
+ * under grant agreement no. 289016.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("universal"),
+    gpii = fluid.registerNamespace("gpii"),
+    jqUnit = fluid.require("jqUnit");
+
+fluid.registerNamespace("gpii.settingsHandlers");
```

Comment:
Should remove these two lines - and instead have

fluid.registerNamespace("gpii.tests.settingsHandlers.webSockets")

and all public definitions in this file should be placed in that namespace